plugins/github-pull-requests-board/src/utils/functions.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Entity } from '@backstage/catalog-model';
+import moment from 'moment';
+import { Reviews, PullRequests, ReviewDecision, PullRequestsColumn, Author } from './types';
+import { COLUMNS } from './constants';
+
+const GITHUB_PULL_REQUESTS_ANNOTATION = 'github.com/project-slug';
+
+
+export const getProjectNameFromEntity = (entity: Entity): string => {
+  return entity?.metadata.annotations?.[GITHUB_PULL_REQUESTS_ANNOTATION] ?? '';
+};
+
+export const getApprovedReviews = (reviews: Reviews = []): Reviews => {
+  return reviews.filter(({ state }) => state === 'APPROVED');
+};
+
+export const getCommentedReviews = (reviews: Reviews = []): Reviews => {
+  return reviews.filter(({ state }) => state === 'COMMENTED');
+};
+export const getChangeRequests = (reviews: Reviews = []): Reviews => {
+  return reviews.filter(({ state }) => state === 'CHANGES_REQUESTED');
+};
+
+export const filterSameUser = (users: Author[]): Author[] => {
+  return users.reduce((acc, curr) => {
+    const contaisUser = acc.find(({ login }) => login === curr.login);
```

Comment:
I suspect it is a typo
```suggestion
    const containsUser = acc.find(({ login }) => login === curr.login);
```